cmd/libsnap-confine-private/classic.c

```diff
@@ -59,5 +59,7 @@ sc_distro sc_classify_distro(void)
 
 bool sc_should_use_normal_mode(sc_distro distro, const char *base_snap_name)
 {
-	return distro != SC_DISTRO_CORE16 || !sc_streq(base_snap_name, "core");
+	return distro != SC_DISTRO_CORE16 || !(sc_streq(base_snap_name, "core")
+					       || sc_streq(base_snap_name,
+							   "ubuntu-core"));
```

Comment:
as I said elsewhere I don't think we need this change if we did things in the other order,  I'm also not sure why this is a function here (to enable having tests for it?), I see similar code doing checks on distro values in snap-confine that is not wrapped in one line functions in the lib,  also this file name seems not completely what it's about

Reply:
The change has been dropped now.